llmware/library.py

```diff
@@ -393,15 +507,44 @@ def set_knowledge_graph_status (self, status_message):
     def get_and_increment_doc_id(self):
 
         """ Convenience method in library class - mirrors method in LibraryCatalog - increments, tracks and provides a
```

Comment:
Delete space before you start the doc .
```suggestion
        """Convenience method in library class - mirrors method in LibraryCatalog - increments, tracks and provides a
```